package.json

```diff
@@ -10,6 +10,10 @@
   "bugs": {
     "url": "https://github.com/getsentry/sentry-docs/issues"
   },
+  "private": true,
+  "workspaces": [
+    "./packages/*"
+  ],
```

Comment:
l: For our target architecture we could think of moving changelog, user-docs and dev-docs to `./apps/*` and all shared code (UI, tsconfig, eslint, tailwind,...) to `./packages`